docs/source/proposal-template.md

```diff
@@ -1,7 +1,7 @@
-[LICENSE](https://github.com/ESIPFed/gsoc/blob/master/LICENSE) (This template was adapted from [ESIP](https://www.esipfed.org/)'s organization [template](https://github.com/ESIPFed/gsoc/blob/master/STUDENT-proposal-template.md): thank you ESIP!)
-
 # Title
 
+[LICENSE](https://github.com/ESIPFed/gsoc/blob/master/LICENSE) (This template was adapted from [ESIP](https://www.esipfed.org/)'s organization [template](https://github.com/ESIPFed/gsoc/blob/master/STUDENT-proposal-template.md): thank you ESIP!)
```

Comment:
I'm afraid that those trying to propose might put this directly like this. Can you somehow ignore this line in linting? Thanks

Reply:
That makes sense @lsetiawan. From my understanding, it seems like the `.markdownlintignore` isn't as advanced as `.gitignore`. It looks like it can only ignore files and folders. Since we have already applied markdownlint to this file and it looks like we wont  change it, I say we just ignore this file when linting. I will go ahead and add a `.markdownlintignore` and change the position of the sentence.